packages/openapi-react-query/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "openapi-react-query",
   "description": "Fast, type-safe @tanstack/react-query client to work with your OpenAPI schema.",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
@kerwanp this is something Changesets is bad at that you have to monitor—when stacking multiple version bumps of dependentpackages, Changesets has a longstanding bug where it’ll bump the downstream package too high. There are still open issues on this, and it doesn’t really get fixed. But publishing a major version accidentally can get annoying pretty quick.

How I resolve it is just by pushing to the branch and manually fixing it. It doesn’t frequently enough where I’ve invested time to try and fix it in Changesets, but every now and then I do have to manually override it.